packages/jellyfish-transaction/src/script/defi/dftx_price.ts

```diff
@@ -0,0 +1,59 @@
+import BigNumber from 'bignumber.js'
```

Comment:
Perhaps should test `dftx_price` composable individually, the interfaces looks very generic and applicable for different usage.

Reply:
it's only used for OracleTx, it's a very specific structure